components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -53,7 +53,7 @@ public class ServiceProvider implements Serializable {
     private static final String IS_MANAGEMENT_APP = "IsManagementApp";
 
     private static final String IS_B2B_SELF_SERVICE_APP = "IsB2BSelfServiceApp";
-    private static final String IS_APPLICATION_ACCESS_ENABLED = "applicationEnabled";
+    private static final String IS_APPLICATION_ENABLED = "applicationEnabled";
```

Comment:
```suggestion
    private static final String IS_APPLICATION_ENABLED = "IsApplicationEnabled";
```
lets follow the same pattern as others (ex: IS_B2B_SELF_SERVICE_APP, IS_MANAGEMENT_APP )

Reply:
This will lead to a change at API server as well.

Reply:
IMO we should have maintain consistency otherwise overtime the quality of the overall code could start to degrade. Since the API is available in a released IS version and is not shared lets change the api layer as needed.

Reply:
Also why is a change in the core require a change in the API?

Reply:
It was suggested maintain 'Better if we can have the similar name in API and backend service.' , though it does not necessitate.